components/ProposalCard.tsx

```diff
@@ -127,7 +127,7 @@ export const RenderProposalContent = ({
           token_id,
         }),
       };
-    } else {
+    } else if (metadata.meta) {
```

Comment:
In which case, we will have `meta` in `metadata`?

Reply:
Tbh I don't know. I had a crash with one of my wallets, so I just logged the data and handled the case to avoid crashes. It may be some legacy of the previous contract version